### PR TITLE
chore: renku-graph upgraded to 2.44.0

### PR DESCRIPTION
**Pull request by RenkuBot**
This is an automated pull request.

/deploy

### CODE REVIEW

File: helm-chart/renku/values.yaml [hunk 1]
```diff
@@ -22,7 +22,6 @@ global:
     # url:
```
Comment (aledegano) [inline helm-chart/renku/values.yaml:22]: @jachro an unruly formatter hit again!

Comment (jachro) [reply helm-chart/renku/values.yaml:22]: That's not me :) I didn't even open the file.
But I'll fix it.

Comment (jachro) [reply helm-chart/renku/values.yaml:22]: Actually, it happened in this commit `ae26f19856385c6880d8266fae361dc2d6481b35` which was done by the Bot. I'm guessing the script updating the version might be doing some reformatting.

Comment (aledegano) [reply helm-chart/renku/values.yaml]: ah, yes, sorry, now I remember. It's something to do with `yq`... I think the upgrade should fix it and it's in my TODO...

Comment (jachro) [reply helm-chart/renku/values.yaml]: @aledegano I fixed it.

Comment (aledegano) [reply helm-chart/renku/values.yaml]: Thanks Kuba!
I'll check the problem with the script.

**Review by aledegano — APPROVED** [review]

**Review by lokijuhy — APPROVED** [review]

**Pull request merged**